aws-csharp/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Aws.S3;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Deployment.RunAsync(() => {
+
+            // Create an AWS resource (S3 Bucket)
+            var bucket = new Bucket("my-bucket");
+
+            // Export the name of the bucket
+            return new Dictionary<string, object>
+            {
+                { "bucketName", bucket.Id },
+            };
+        });
+    }
+}
```

Comment:
Super nitpick: Any reason not to have a trailing newline on all of these?

Reply:
Probably Visual Studio not adding them... Added in several files.